routes/duration.js

```diff
@@ -1,18 +1,8 @@
 const express = require('express'); 
 const router = express.Router();
-const repsCtrl = require('../controllers/duration');
+const durationCtrl = require('../controllers/duration');
 
-router.get("/", function (req, res) {
-    res.send("index", {
-      user: req.user,
-    });
-  });
+router.get('/', durationCtrl.index); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
router.get("/", durationCtrl.index); 
```